src/math_ops.jl

```diff
@@ -346,6 +346,26 @@ function rotation_tensor(θ::Number)
     return Tensor{2, 2}((c, s, -s, c))
 end
 
+@doc raw"""
+    rotation_tensor(ψ::Number,θ::Number,ϕ::Number)
+
```

Comment:
While I appreciate the feature, can we leave a note for the user that this might be hamful to use in specific scenarios (due to the possibility to run into gimbal locking)?

Reply:
I think this should be here because from my experience many users are not aware of this.